test_py/test_make_request.py

```diff
@@ -0,0 +1,253 @@
+import json
+
+import pytest
+import requests
+from src_py.grndwork_api_client.dtos import ContentRange
+from src_py.grndwork_api_client.make_request import (
+    make_paginated_request,
+    make_request,
+    parse_content_range,
+)
+
+
+API_URL = 'https://api.grndwork.com/v1/tokens'
+
+
+@pytest.fixture(name='requests', autouse=True)
+def fixture_requests(mocker):
+    req_mock = mocker.patch(
+        target='src_py.grndwork_api_client.make_request.requests',
+        spec=requests,
+    )
+    response = mocker.MagicMock()
+    response.status_code = 200
+    response.json.return_value = {}
+    response.headers = {'Content-Range': 'items 1-1/1'}
```

Comment:
Not every response from the server is going to include this.

Reply:
Removed.